api/src/modules/import-data/workers/import-data.producer.ts

```diff
@@ -0,0 +1,27 @@
+import { InjectQueue } from '@nestjs/bull';
+import { Injectable } from '@nestjs/common';
+import { Job, Queue } from 'bull';
+import { TinyTypeOf } from 'tiny-types';
+
+export interface ExcelImportJob {
+  xlsxFilePath: string;
+  userId: string;
+}
+
+export class ImportQueueName extends TinyTypeOf<string>() {}
+
+export const importQueueName: ImportQueueName = new ImportQueueName(
+  'excel-import',
+);
```

Comment:
Not a very smart use of the recently added tool, just a try. This could be just a common uppercase constant

Reply:
you mean the queue name? use a Symbol

Reply:
I gave it a try, and it does not work, (even after converting to value as a string)

1.  Even tho I saw the marxan approach, the decorators and the BullModule we are using need that value as string
2. After converting the value to a string resulting 'Symbol(queueName)' there is a exception thrown when API starts.
3. Taking in account the constant statement and the primitive nature of the string, this should be inmutable as well

Thoughts?